src/components/dashboard/GovernanceSection/GovernanceSection.tsx

```diff
@@ -22,6 +22,10 @@ const GovernanceSection = () => {
   const claimingApp = claimingSafeApp?.[0]
   const fetchingSafeClaimingApp = !claimingApp && !errorFetchingClaimingSafeApp
 
+  if (!claimingApp) {
```

Comment:
What if it's just being loaded? Or errored out?
We want to show both a loading state and an error.

Reply:
True. I've adjusted it to use the same logic as the token widget.